helper_mysql.conf

```diff
@@ -0,0 +1,11 @@
+agentAddress = "/tmp/agent.sock"
```

Comment:
I'll add 
```
# MySQL expect intermediate certificates inside `svidBundleFile` file
# instead of svidFile
mergeCAWithIntermediates = true
```
inside `examples/mysql/helper.conf`